demo/src/source-info.js

```diff
@@ -3,19 +3,15 @@ export const source = {
   width: 206,
   tileSize: 256,
   // channels: {
-  //   channel_0: "https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/single_channel_pyramid/img_pyramid/channel_0",
-  //   channel_1: "https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/single_channel_pyramid/img_pyramid/channel_1",
-  //   channel_2: "https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/single_channel_pyramid/img_pyramid/channel_2",
-  //   channel_3: "https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/single_channel_pyramid/img_pyramid/channel_3",
+  //   "Cy3 - Synaptopodin (glomerular)": "https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/single_channel_pyramid/img_pyramid/channel_0",
+  //   "Cy5 - THP (thick limb)": "https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/single_channel_pyramid/img_pyramid/channel_1",
+  //   "DAPI - Hoescht (nuclei)": "https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/single_channel_pyramid/img_pyramid/channel_2",
+  //   "FITC - Laminin (basement membrane)": "https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/single_channel_pyramid/img_pyramid/channel_3",
   // },
   channels: {
-    channel_0:
-      'https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/tiff-pyramid/vanderbilt_test_0.ome.tiff',
-    channel_1:
-      'https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/tiff-pyramid/vanderbilt_test_1.ome.tiff',
-    channel_2:
-      'https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/tiff-pyramid/vanderbilt_test_2.ome.tiff',
-    channel_3:
-      'https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/vanderbilt-data/tiff-pyramid/vanderbilt_test_3.ome.tiff'
+    "Cy3 - Synaptopodin (glomerular)": "https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/VAN0001-RK-1-21_24-MxIF-mxIF_toIMS/vanderbilt_test_Cy3 - Synaptopodin (glomerular).ome.tiff",
+    "Cy5 - THP (thick limb)": "https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/VAN0001-RK-1-21_24-MxIF-mxIF_toIMS/vanderbilt_test_Cy5 - THP (thick limb).ome.tiff",
+    "DAPI - Hoescht (nuclei)": "https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/VAN0001-RK-1-21_24-MxIF-mxIF_toIMS/vanderbilt_test_DAPI - Hoescht (nuclei).ome.tiff",
+    "FITC - Laminin (basement membrane)": "https://vitessce-vanderbilt-data.storage.googleapis.com/test-data/VAN0001-RK-1-21_24-MxIF-mxIF_toIMS/vanderbilt_test_FITC - Laminin (basement membrane).ome.tiff",
```

Comment:
Factor out the repeated strings? ... particularly since they aren't aligned, it takes some looking to see what is just copy and paste.